host/sgx/sgxquote.c

```diff
@@ -253,6 +248,39 @@ static bool _load_sgx_dcap_qvl(void)
     return (_qvl_module != NULL);
 }
 
+static bool _sgx_use_in_process_quoting()
+{
+    // Before: the in-process call path is used unless the environment variable
+    // SGX_AESM_ADDR is set. Now: the environment variable SGX_AESM_ADDR is
+    // deprecated. In-process and out-of-process call paths are the default
+    // settings for Windows and Linux, respectively, unless the user demands
+    // in-process call path be used by setting the environment variable
+    // SGX_USE_IN_PROCESS_QUOTING to 1.
+    static char* sgx_use_in_process_quoting;
```

Comment:
No need to be static because you retrieve it every time the function is called. Give it some buffer size because you want to know its value. i.e.

char sgx_use_in_process_quoting[2];